microsetta_interface/templates/source.jinja2

```diff
@@ -245,72 +245,81 @@
-              <div class="row">
-                <div class="col-sm">
-                  {{survey.survey_template_title |e}}
-                </div>
-                <div class="col-sm">
-                    {% if survey.answered %}
-                    <span class="fa fa-check"/>
-                    {% else %}
-                    <span class="fa fa-unchecked"/>
-                    {% endif %}
-                </div>
-              </div>
-            </div>
-        </a>
-        {% endfor %}
-    </div>
-    {% endif %}
-    
+
     {% if available_surveys|length > 0 %}
     <hr style="border-top: dotted 1px"/>
     <p>{{ _('Available optional surveys') }}:</p>
     <div class="list-group">
         <div class="container">
             <div class="row">
-                <div class="col-sm">
+                <div class="col-3">
                     <i>{{ _('Survey Title') }}</i>
                 </div>
-                <div class="col-sm">
+                <div class="col-8">
                     <i>{{ _('Survey Description') }}</i>
                 </div>
+                <div class="col">
+                    &nbsp;
+                </div>
             </div>
         </div>
         {% for detail in available_surveys %}
         <div class="container list-group-item {{loop.cycle('odd', 'even') }}">
             <div class="row">
-                <div class="col-sm">
+                <div class="col-3">
+                    {{ detail.survey_template_title|e }}
+                </div>
+                <div class="col-8">
+                    {{ detail.description }}
+                </div>
+                <div class="col">
                     {% if detail.new_tab %}
-                    <a class="btn btn-outline-info" href="/accounts/{{ account_id }}/sources/{{ source_id }}/take_survey?survey_template_id={{ detail.survey_template_id }}" target="_blank">
+                    <a class="btn btn-primary" href="/accounts/{{ account_id }}/sources/{{ source_id }}/take_survey?survey_template_id={{ detail.survey_template_id }}" target="_blank">
                     {% else %}
-                    <a class="btn btn-outline-info" href="/accounts/{{ account_id }}/sources/{{ source_id }}/take_survey?survey_template_id={{ detail.survey_template_id }}">
+                    <a class="btn btn-primary" href="/accounts/{{ account_id }}/sources/{{ source_id }}/take_survey?survey_template_id={{ detail.survey_template_id }}">
                     {% endif %}
-                    {{ detail.survey_template_title|e }}
+                        Start
                     </a>
                 </div>
+            </div>
+        </div>
+        {% endfor %}
+    </div>
+    <br />
+    {% endif %}
+
+    {% if surveys|length > 0 %}
+    <hr style="border-top: dotted 1px"/>
+    <p>{{ _('Completed surveys') }}:</p>
+    <div class="list-group">
+        {% for survey in surveys %}
+        <a>
```

Comment:
Why wrap this in `a`?

Reply:
Honestly, no idea, I just moved the entire block for completed surveys and changed "Surveys taken" to "Completed surveys." Scrolling back through the history of the file, it appears the tag has been there for years. Want me to remove it?

Reply:
I think it would be correct to remove as we don't have my hyperlinks in that block, thanks!